src/annotations.jl

```diff
@@ -92,3 +98,5 @@ function merge_overlapping_annotations(annotations)
     end
     return merged
 end
+
+merge_overlapping_annotations(annotations) = merge_overlapping_annotations(TimeSpans.overlaps, annotations)
```

Comment:
`overlaps` is exported, no need for the qualification
```suggestion
merge_overlapping_annotations(annotations) = merge_overlapping_annotations(overlaps, annotations)
```